build.sbt

```
@@ -8,7 +8,7 @@ val circeVersion = "0.13.0"
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.6.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.19"
+val cogniteSdkVersion = "1.5.20-SNAPSHOT"
```

```suggestion
val cogniteSdkVersion = "1.5.20"
```
Once https://github.com/cognitedata/cognite-sdk-scala/pull/432 is merged